include/radix_tree/radix_tree.hpp

```diff
@@ -0,0 +1,557 @@
+#ifndef RADIX_TREE_HPP
```

Comment:
This is a copyright violation without appropriate attribution as required by the upstream library.

Reply:
Thanks. Fixed.